src/main/java/org/radarbase/appserver/controller/UploadController.java

```diff
@@ -62,6 +63,12 @@ public class UploadController {
       "/" + PathsUtil.FILE_PATH +
       "/" + PathsUtil.TOPIC_PATH + "/" + PathsUtil.TOPIC_ID_CONSTANT +
       "/upload")
+  @CrossOrigin(
+      origins = "*",
+      allowedHeaders = "*",
+      exposedHeaders = "Location", // needed to get the URI of the uploaded file in aRMT
+      methods = { RequestMethod.POST }
```

Comment:
Since the rest are already specified in the `MultiHttpSecurityConfig`, should we only use `exposedHeaders` here?